src/sonic-py-common/sonic_py_common/multi_asic.py

```diff
@@ -487,3 +492,21 @@ def get_asic_presence_list():
         # This is not multi-asic, all asics should be present.
         asics_list = list(range(0, get_num_asics()))
     return asics_list
+
+
+def is_front_panel_port(port, role=None):
```

Comment:
@vivekrnv I don't see this used anywhere. Is this going to be used other place?

Reply:
@prgeor  This API is used in other repositories. Example https://github.com/sonic-net/sonic-platform-daemons/pull/455/